src/app/FakeLib/CMakeHelper.fs

```diff
@@ -0,0 +1,286 @@
+﻿[<AutoOpen>]
```

Comment:
OK, I was not aware that some modules are not auto-opened because of their low-usage... As FAKE scripts tends to hide as most F# specifics as possible (_"It is using an easy domain-specific language (DSL) so that you can start using it without learning F#."_), I believed that removing the need for specifying any module name was a design choice.

As not auto-opening `CMakeHelper` forces end-users to explicitly use the module name at each function call, I would like to rename:
* `CMakeHelper` into `CMake`
* `CMakeGenerate` into `Generate`
* `CMakeBuild` into `Build`

Usages of the helper would be cleaner because of their `<Noun>.<Verb>` form.

Are you OK with this?